### PR TITLE
Breadcrumb

### PR DESCRIPTION
**Pull request by alyssadee**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/components/Breadcrumb/index.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        'brew-Breadcrumb',
+        className,
+    );
+
+
+    return (
+        <div className={classes}>
+            {crumbs.map((crumb, index) => {
+                const notLast = crumbCount !== index;
+                return (
```
Comment (emdrayallday) [inline src/components/Breadcrumb/index.tsx]: Maybe add line spacing like we do regularly? 2 spaces before/after each section of content.

File: theme/cerveza/src/components/_Breadcrumb.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
+@import "../utilities";
+
+
+// ==============================================
+// General
+// ==============================================
+
+.brew-Breadcrumb--link {
```
Comment (emdrayallday) [inline theme/cerveza/src/components/_Breadcrumb.scss]: Maybe nest these?

`.brew-Breadcrumb {
    &--separator{}
    &--custom{}
}`

File: src/components/Breadcrumb/index.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                    <BrowserRouter key={index}>
+                        <Typography variant="body2">
+                            {crumb.url && notLast ?
+                                <Link className="brew-Breadcrumb--link" to={crumb.url}>{crumb.title}</Link> :
+                                <Typography variant="body2">{crumb.title}</Typography>
+                            }
```
Comment (emdrayallday) [inline src/components/Breadcrumb/index.tsx]: Should the ternary lines be more like this

```
crumb.url && notLast
    ? true
    : false
```
I feel like it's a little more readable

Comment (lawren) [reply src/components/Breadcrumb/index.tsx]: I second this.

File: theme/cerveza/src/components/_Breadcrumb.scss (hunk 2 above)
Comment (lawren) [reply theme/cerveza/src/components/_Breadcrumb.scss]: Seconded.

**Pull request merged**